cpp/src/Slice/Parser.cpp

```diff
@@ -2433,8 +2433,8 @@ Slice::ClassDef::createDataMember(
     const TypePtr& type,
     bool isOptional,
     int tag,
-    const SyntaxTreeBasePtr& defaultValueType,
-    const string& defaultValueString)
+    SyntaxTreeBasePtr defaultValueType,
```

Comment:
All this duplicated code in the 3 createDataMember functions is unfortunate. I didn't fix it.